README.md

```diff
@@ -180,7 +180,7 @@ Step 2. Launch a redis container
 ```bash
 docker run --name gitlab-redis -d \
     --volume /srv/docker/gitlab/redis:/var/lib/redis \
```

Comment:
according to https://hub.docker.com/_/redis the volume should be mounted at `/data`, so this line would need to be updated

Reply:
Would the volume in the `docker-compose.yml` also need to be modified:

https://github.com/sameersbn/docker-gitlab/blob/09b8c1c265785fcf2e872bda6fcb6709f55b3e4d/docker-compose.yml#L3-L10

```
```

Reply:
yea, i believe so.
there's also a reference in the Makefile